python/Ganga/Runtime/bootstrap.py

```diff
@@ -583,80 +574,23 @@ def set_cmdline_config_options(sects=None):
         GangaRootPath = os.path.normpath(os.path.join(os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe()))), '../..'))
         self.options.config_path = Ganga.Utility.files.expandfilename(os.path.join(GangaRootPath, self.options.config_path))
 
-        try:
-            hostname = Ganga.Utility.util.hostname()
-        except Exception as x:  # fixme: use OSError instead?
-            hostname = 'localhost'
-
-        # the system variables (such as VERSION) are put to DEFAULTS section of the config module
-        # so you can refer to them in the config file
-        # additionally they will be visible in the (write protected) [System]
-        # config module
-        syscfg = Ganga.Utility.Config.makeConfig('System', "parameters of this ganga session (read-only)", cfile=False)
-        syscfg.addOption('GANGA_VERSION', _gangaVersion, '')
-        syscfg.addOption('GANGA_PYTHONPATH', _gangaPythonPath, 'location of the ganga core packages')
-        syscfg.addOption('GANGA_CONFIG_PATH', self.options.config_path,'site/group specific configuration files as specified by --config-path or GANGA_CONFIG_PATH variable')
-        syscfg.addOption('GANGA_CONFIG_FILE', self.options.config_file, 'current user config file used')
-        syscfg.addOption('GANGA_HOSTNAME', hostname, 'local hostname where ganga is running')
+        from Ganga.Utility.Config import getConfig
```

Comment:
Is this not already imported at the top of the file? One of them is `from Ganga.Utility.Config.Config` and the other is `from Ganga.Utility.Config` so perhaps that is why it wasn't noticed.